apps/vanilla/src/pages/exercise2/scripts/models/TurnOrder.ts

```diff
@@ -0,0 +1,9 @@
+/** Data about who is turn. */
+export interface TurnOrder {
+
+	/** Current player's index. */
+	readonly currentPlayerIndex: number;
```

Comment:
At the moment you add `Player` model, maybe you can use it in this, or use `Player['id']` (at moment review this field have `index` name)?

Reply:
Do you mean renaming?
```suggestion
	readonly currentPlayerId: Player['id'];
```